### DISCUSSION TITLE
Woox R7052 Smart SOS Button (seen as unsupported TS0215A)

### DISCUSSION THREAD

**Discussion by adynis** [Koenkk/zigbee2mqtt#20496] [Devices]
I use zigbee2mqtt as part of HomeAssistant.
I have this device: https://wooxhome.com/woox-r7052-smart-sos-button-p50
![image](https://github.com/Koenkk/zigbee2mqtt/assets/10837368/7e17d706-8f55-43d0-a340-a6f6d3bc6ec6)
And it is identified as:
Unsupported  -  TS0215A  -  _TZ3000_ssp0maqm

But I've seen in last z2m update that this is properly recognized: https://www.zigbee2mqtt.io/devices/TS0215A_sos.html

My unsupported device is also an SOS button ... can i link it somehow to TS0215A_sos ? (PS. I don't have experience with external converters)

**Comment by Koenkk** [comment]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.
